packages/p2p/src/components/modal-manager/modal-manager-context-provider.jsx

```diff
@@ -46,7 +60,6 @@ const ModalManagerContextProvider = props => {
         setIsModalOpen(true);
     };
     const hideModal = (should_save_form_history = false) => {
-        modal_props.delete(active_modal.key);
```

Comment:
Refactored out this to use `registered_modals` since its not correct to de-register the modal props when the modal is hidden, because it could be shown again and would need the registered props later